modules/client/test/integration/client/methods.test.ts

```diff
@@ -68,41 +68,6 @@ describe("Client", () => {
     });
 
     describe("DAO Creation", () => {
-      it("Should estimate gas fees for creating a DAO", async () => {
-        const context = new Context(contextParamsLocalChain);
-        const client = new Client(context);
-
-        const daoName = "ERC20VotingDAO_" +
-          Math.floor(Random.getFloat() * 9999) + 1;
```

Comment:
Why are we removing this test, especially now?

Reply:
Is the same test, it was duplicated